Jenkinsfile

```diff
@@ -206,7 +206,12 @@ pipeline {
                     fingerprintArtifacts: true
                 )
                 sh 'kubectl get nodes -o wide'
-                sh "nix-shell --run './scripts/e2e-test.sh --device /dev/sdb --tag \"${env.GIT_COMMIT_SHORT}\" --registry \"${env.REGISTRY}\"'"
+                sh "nix-shell --run './scripts/e2e-test.sh --device /dev/sdb --tag \"${env.GIT_COMMIT_SHORT}\" --registry \"${env.REGISTRY}\" --logsdir ./logs/mayastor --logs '"
+              }
+              post {
+                always {
```

Comment:
I think we want to save logs only in case of an error and not always.

Reply:
Capture  logs always and dispose regularly is more useful approach.